src/components/select-utils.js

```diff
@@ -63,23 +63,23 @@ export const joinPostData = state => postId => {
 
   // Get the list of post's recipients
   const recipients = post.postedTo
-    .map((subscriptionId) => {
-      let userId = (state.subscriptions[subscriptionId]||{}).user
-      let subscriptionType = (state.subscriptions[subscriptionId]||{}).name
-      if (userId === post.createdBy && subscriptionType === 'Directs') {
-        // Remove "directs to yourself" from the list
-        return false
-      } else {
-        return userId
-      }
+    .map(subscriptionId => {
+      const userId = (state.subscriptions[subscriptionId]||{}).user
+      const subscriptionType = (state.subscriptions[subscriptionId]||{}).name
+      return userId !== post.createdBy || subscriptionType !== 'Directs' ? userId : false
```

Comment:
Don't know really, it was 'before my patch'. As a side note, I'm always 'for' removing comments and unnecessary code.